wforce/wforce-replication.hh

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of PowerDNS or weakforced.
+ * Copyright -- PowerDNS.COM B.V. and its contributors
```

Comment:
PowerDNS.COM B.V. is still legit here?

Reply:
Yeah still legit. That's who I work for.